src/modules/Thread/components/ParentMessageInfo/ParentMessageInfoItem.tsx

```diff
@@ -91,7 +91,7 @@ export default function ParentMessageInfoItem({
     return tokenizeMessage({
       messageText: (message as UserMessage)?.message,
     });
-  }, [message?.updatedAt]);
+  }, [message?.updatedAt, (message as UserMessage)?.message]);
```

Comment:
is it always user message?

Reply:
Probably not. I did this for typing. See 92 line too.

Reply:
Can we please add comment so people don't get confused? same for all others below.